Paintroid/src/main/java/org/catrobat/paintroid/tools/helper/CropAlgorithm.kt

```diff
@@ -16,12 +16,11 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.paintroid.tools.helper
 
-package org.catrobat.paintroid.tools.helper;
+import android.graphics.Bitmap
+import android.graphics.Rect
 
-import android.graphics.Bitmap;
-import android.graphics.Rect;
-
-public interface CropAlgorithm {
-	Rect crop(Bitmap bitmap);
+interface CropAlgorithm {
+    fun crop(bitmap: Bitmap?): Rect?
```

Comment:
```suggestion
    fun crop(bitmap: Bitmap): Rect?
```
Look for usages please

Reply:
I had checked it, the bitmap can be nullable, so `?` should be present.